CMakeLists.txt

```diff
@@ -39,16 +39,25 @@ else()
   set (EXTRA_TEST_LIB_DEPS)
 endif()
 
+set(skip_pybind11_default_value OFF)
+if (MSVC)
+  set(skip_pybind11_default_value ON)
+endif()
+
+option(SKIP_PYBIND11
+      "Skip generating Python bindings via pybind11"
+      ${skip_pybind11_default_value})
+
 include(test/find_dri.cmake)
 FindDRI()
 
 option(USE_SYSTEM_PATHS_FOR_PYTHON_INSTALLATION
       "Install python modules in standard system paths in the system"
-      OFF)
+      ${skip_pybind11_default_value})
```

Comment:
Not sure if this setting should depend on `skip_pybind11_default_value`. Maybe it should depend on `SKIP_PYBIND11`, but I'm not even sure  of that. Maybe best to leave it unchanged.